src/hyperlight_host/src/hypervisor/regs/fpu.rs

```diff
@@ -0,0 +1,466 @@
+/*
+Copyright 2024 The Hyperlight Authors.
```

Comment:
I am not sure what the guidance is here, but shouldn't we use `2025` if we added the file this year? I'm asking for myself also, to know how I do it.

Reply:
For new files , I think definitely, I think i we update files we should probably update it as well?